crates/ra_ide/src/completion/complete_snippet.rs

```diff
@@ -33,6 +33,24 @@ pub(super) fn complete_item_snippet(acc: &mut Completions, ctx: &CompletionConte
         None => return,
     };
 
+    snippet(
+        ctx,
+        cap,
+        "Test module",
+        "\
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn ${1:test_name}() {
+        $0
```

Comment:
Should $0 and $1 be swapped.

Reply:
No in fact $1 will be the first cursor position after confirm snippet